lib/grammers-client/src/types/callback_query.rs

```diff
@@ -44,6 +45,30 @@ impl CallbackQuery {
             raw: query,
             client: client.clone(),
             chats: chats.clone(),
+            inline_msg_id: None,
+        }
+    }
+
+    pub fn from_inline_raw(
+        client: &Client,
+        query: tl::types::UpdateInlineBotCallbackQuery,
+        chats: &Arc<types::ChatMap>,
+    ) -> Self {
+        Self {
+            raw: tl::types::UpdateBotCallbackQuery {
```

Comment:
Not sure how I feel about "lying" about the original raw type (but I can't remember if we do it elsewhere, and updates are quite similar, so this is OK).

Reply:
ok